.github/workflows/ci.yml

```diff
@@ -58,7 +58,7 @@ jobs:
       #     root: _site
 
       - name: HTML Proofer
-        run: bundle exec htmlproofer ./_site
+        run: bundle exec htmlproofer --ignore-urls "/mermaid.ink/" ./_site
```

Comment:
Why do we need to ignore them?

Reply:
Htmlproof has an issue with the alt image, in the mermaid we can't set alt for svg.